scripts/add.tokens.erc20.mjs

```diff
@@ -0,0 +1,264 @@
+	try {
+		const html = await fetchHtml(url);
+		const $ = load(html);
+
+		const tokens = [];
+		$('#supported-ckerc20-tokens + table tbody tr').each((index, element) => {
+			const symbol = $(element).find('td').eq(0).text().trim();
+			const contractAddress = $(element).find('td').eq(2).text().trim();
+			tokens.push({ symbol, contractAddress });
+		});
+
+		console.log(
+			`Found ${tokens.length} tokens on ${url}: ${tokens.map((t) => t.symbol).join(', ')}`
+		);
+
+		return tokens;
+	} catch (err) {
+		console.error(`Error fetching supported tokens on ${url}:\n${err}`);
+		process.exit(1);
+	}
+};
+
+const main = async () => {
+	const prodTokens = await fetchSupportedTokens(PROD_DASHBOARD_URL);
+	const testnetTokens = await fetchSupportedTokens(TESTNET_DASHBOARD_URL);
+
+	const testnetTokenMap = new Map(
+		testnetTokens.map((token) => [token.symbol.replace('ckSepolia', 'ck'), token.contractAddress])
+	);
+
+	const tokensToProcess = [];
+
+	for (const token of prodTokens) {
+		const testnetContractAddress = testnetTokenMap.get(token.symbol);
+		if (!testnetContractAddress) {
+			continue;
+		}
+
+		const tokenName = token.symbol.slice(2);
+		const tokenFilePath = path.join(SRC_PATH, `tokens.${tokenName.toLowerCase()}.env.ts`);
+		try {
+			await fs.access(tokenFilePath);
+		} catch (err) {
+			tokensToProcess.push({
+				tokenName,
+				symbol: token.symbol,
+				contractAddress: token.contractAddress,
+				testnetContractAddress
+			});
+		}
+	}
+
+	if (tokensToProcess.length === 0) {
+		console.log('All prod tokens that have a test token are set.');
+		return;
+	}
+
+	for (const { tokenName, contractAddress, testnetContractAddress } of tokensToProcess) {
+		console.log('--------------------------------');
+		console.log(`Creating file for token ${tokenName}`);
+		const tokenDetails = await fetchTokenDetails(contractAddress);
+
+		const { newFileName, mainnetToken, testnetToken } = await createEnvFile(
+			tokenName,
+			tokenDetails,
+			contractAddress,
+			testnetContractAddress
+		);
+
+		await updateTokensErc20Env(newFileName, mainnetToken, testnetToken);
+
+		console.log(`Finished for token ${tokenName}`);
+	}
+
+	console.log('--------------------------------');
+	console.log(
+		'Final Step: To complete the integration of the new tokens, you need to create SVG icon files for each token and place them in the correct directory.'
+	);
+	console.log(`Navigate to the following directory: ${path.join(SRC_DIR)}`);
+
+	tokensToProcess.forEach(({ tokenName }) => {
+		console.log(
+			`- For token ${tokenName}, create an SVG file named '${tokenName.toLowerCase()}.svg'`
+		);
+	});
+
+	console.log(
+		'Ensure that the SVG files are valid SVG format and represent the tokens, typically the logo or symbol of each token.'
+	);
+
+	tokensToProcess.forEach(({ tokenName }) => {
+		console.log(
+			`Example: For token ${tokenName}, the SVG file should be: ${SRC_DIR}/${tokenName.toLowerCase()}.svg`
+		);
+	});
+};
+
+main().catch(console.error);
```

Comment:
for what it's worth, top level `await` is possible

Reply:
done